addons/point_of_sale/tools/posbox/configuration/odoo.conf

```diff
@@ -7,3 +7,4 @@ pidfile = /var/run/odoo/odoo.pid
 limit_time_cpu = 600
```

Comment:
should not we remove or increase the values of the `limit_time` ? Otherwise the threads of the driver might crash as they are meant to run endlessly

Reply:
indeed my bad :) thanks for the info!